docs/guides/faq.md

```diff
@@ -0,0 +1,120 @@
+the [pull request template](/.github/PULL_REQUEST_TEMPLATE.md).
+
+### If you don't think you can fix the issue or add the feature
+Open an [issue on the video.js repo](https://github.com/videojs/video.js/issues). Make
+sure that you follow the [issue template](/.github/ISSUE_TEMPLATE.md) and the
+[contributing guide](/CONTRIBUTING.md#filing-issues) so that we can better assist you
+with your issue.
+
+## Q: What is a [reduced test case](https://css-tricks.com/reduced-test-cases/)?
+A [reduced test case](https://css-tricks.com/reduced-test-cases/) is an example of the
+problem that you are facing in isolation. Think of it as example page that reproduces
+the issue in the least amount of possible code. We have a [starter example](http://jsbin.com/axedog/edit?html,output)
+for reduced test cases.
+
+## Q: How do a make a plugin for video.js?
+See the [plugin guide](/docs/guides/plugins.md) for information on making a plugin for video.js.
+
+## Q: How do I add a button to video.js?
+See the [button guide](/docs/guides/button.md) for information on adding a button to video.js.
+
+## Q: Where can I find a list of video.js plugins?
+A list of plugins is avialable on [videojs.com](http://videojs.com/plugins).
+
+## Q: How can I get my plugin listend on the website?
+Add the 'videojs-plugin' [keyword to your array in package.json](https://docs.npmjs.com/files/package.json#keywords)
+and publish your package to npm.
+
+## Q: Where can I find a list of video.js skins?
+See the [video.js github wiki](https://github.com/videojs/video.js/wiki/Skins).
+
+## Q: Does video.js work as an audio only player?
+Yes! It can be used to play audio only files in a `<video>` or `<audio>` tag. The
+difference being that the `<audio>` tag will not have a blank display area and the `<video>`
+tag will.
+
+## Q: Does video.js support audio tracks?
+Yes! See the [audio tracks guide](/docs/guides/audio-tracks.md) for information on using audio tracks.
+
+## Q: Does video.js support video tracks?
+The code for video tracks exists but it has not been tested. See the [video tracks guide](/docs/guides/video-tracks.md)
+for more information on using video tracks.
+
+### Q: Does video.js support text tracks (captions, subtitles, etc)?
+Yes! See the [text tracks guide](/docs/guides/text-track.md) for information on using text tracks.
+
+## Q: Does video.js support HLS (HTTP Live streaming) video?
+video.js supports HLS video if the native HTML5 element supports HLS (e.g. Safari, Edge,
+Chrome for Android, and iOS). For browsers without native support see the [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls)
+project which adds support.
+
+## Q: Does video.js support MPEG Dash video?
+video.js itself does not support MPEG DASH, but a project called [videojs-contrib-dash](https://github.com/videojs/videojs-contrib-dash)
+which is maintained by the video.js organization.
+
+## Q: Does video.js support live video?
+Yes! Video.js adds support for live videos.
```

Comment:
The Flash tech supports simple RTMP streams, and [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls) add support for live HLS.